code-studio/src/js/levels/textMatch.js

```diff
@@ -35,8 +35,13 @@ TextMatch.prototype.ready = function () {
     window.getResult = $.proxy(this.getResult, this);
   }
 
-
-  $("#" + this.id + " textarea.response").blur(function () {
+  var textarea = $("#" + this.id + " textarea.response");
+  textarea.blur(function () {
+    if (window.levelGroup && window.levelGroup.answerChangedFn) {
+      window.levelGroup.answerChangedFn(this.levelId, true);
```

Comment:
Yep!  ES6 is now available everywhere*!

*We might have missed a couple of unusual cases, this has already bitten us once in the embedded-blocks.  But it's definitely good-to-go here.